Update Readme.md (#1)
DevExpress-Examples/asp-net-web-forms-grid-highlight-text-in-template: Readme.md

<!-- default badges end -->

# Grid View for ASP.NET Web Forms - How to highlight search text placed inside a DataItem template
<!-- run online -->
**[[Run Online]](https://codecentral.devexpress.com/t222691/)**
<!-- run online end -->

Search results contained in templates are not highlighted. This example demonstrates how to highlight them manually. 

![](grid-with-highlight-in-template.png)

## Implementation Details

In this example, [DataItemTemplate](https://docs.devexpress.com/AspNet/DevExpress.Web.GridViewDataColumn.DataItemTemplate) contains the [ASPxLabel](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxLabel) control. 

```aspx
<dx:GridViewDataTextColumn FieldName="CategoryName" >
    <DataItemTemplate>
        <dx:ASPxImage ID="ASPxImage1" runat="server" ShowLoadingImage="true" Width="50" ImageUrl='<%# string.Format("~/Images/{0}.jpg", Eval("CategoryID")) %>' />
        <dx:ASPxLabel ID="label" runat="server" Text='<%# Eval("CategoryName") %>' EncodeHtml="false" OnDataBound="label_DataBound" />
    </DataItemTemplate>
</dx:GridViewDataTextColumn>
```

> The `ASPxLabel.DataBound` event handler looking for the search text and wraps the found text with the `<span>` tag with the `dxgvHL` CSS class specified.

Reviewer: ```suggestion
The `ASPxLabel.DataBound` event handler looks for the search text and wraps the found text with the `<span>` tag with the `dxgvHL` CSS class specified.
```
`with the `<span>` tag with the `dxgvHL` CSS class specified` 'with' 'with' causes cofusion

Author: ```suggestion
The `ASPxLabel.DataBound` event handler looks for the search text and applies the `dxgvHL` CSS class to the found text.
```
